### PR TITLE
feat: [SRM-12017]: Extended duration for verifyStep duration

### PR DESCRIPTION
**Pull request by deepesh-ui**
### Summary

Changes for Increase Duration

#### Screenshots

Screenshot 2022-09-13 at 8 36 42 PM

Screenshot 2022-09-14 at 9 14 35 AM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 8 Issues 
* Code issue 8 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11214)

File: src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/ContinousVerificationInputSetStep.tsx [hunk 1]
```diff
@@ -89,6 +92,9 @@ export function ContinousVerificationInputSetStep(
     return { serviceIdentifier: serviceIdentifierData, envIdentifier: envIdentifierData }
   }, [pipeline, formik])
 
+  const extendedDurationFlag = useFeatureFlag(FeatureFlag.ENABLE_VERIFY_STEP_LONG_DURATION)
+  const durationList = extendedDurationFlag ? [...durationOptions, ...extendedDurationOptions] : durationOptions
```
Comment (harshilbits) [inline src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/ContinousVerificationInputSetStep.tsx:96]: This logic is used twice. Please move it to a function and reuse.

Comment (deepesh-ui) [reply src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/ContinousVerificationInputSetStep.tsx]: done

**Review by harshilbits — APPROVED** [review]

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/DeploymentProgressAndNodes.tsx [hunk 2]
```diff
@@ -18,6 +18,19 @@ import VerificationStatusCard from './components/VerificationStatusCard/Verifica
 import type { DeploymentNodeAnalysisResult } from './components/DeploymentNodes/DeploymentNodes.constants'
 import css from './DeploymentProgressAndNodes.module.scss'
 
+const DurationView = ({ durationMs }: { durationMs?: number }): JSX.Element => {
```
Comment (praneshg239) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/DeploymentProgressAndNodes.tsx:21]: Please add the component in new file

**Review by praneshg239 — COMMENTED** [review]
Please check a comment

**Review by praneshg239 — APPROVED** [review]

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/DeploymentProgressAndNodes.tsx (hunk 2 above)
Comment (deepesh-ui) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/DeploymentProgressAndNodes.tsx]: done

**Review by harshilbits — DISMISSED** [review]

**Review by praneshg239 — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Pull request merged**